test/functionality/util/arguments.test.ts

```diff
@@ -1,4 +1,4 @@
-import { splitAtEscapeSensitive } from '../../../src/util/args';
+import { splitAtEscapeSensitive } from '../../../src/util/text/args';
```

Comment:
It is hard to say this way and probably just a separate issue but some of these files should be definitely split into multiple sub-files and maybe even standalone supporting libraries for some functionality. What do you think?

Reply:
#1608